docs/user-guide/zoom.ipynb

```diff
@@ -311,7 +292,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "func = pol_workflow.compute(pol.he3.He3TransmissionFunction[pol.Analyzer])"
+    "func = pol_workflow.compute(pol.TransmissionFunction[pol.Analyzer])"
    ]
   },
   {
```

Comment:
Changes look fine but it seems the notebook is not run in the documentation build (in the built docs from CI run at least).

Reply:
Yes, sorry, I should have been clearer: The notebook requires Mantid, so it does not run in the actually docs build on CI.

Reply:
And I guess you don't want to install Mantid because you want to use `tox` and `pip`...